modules/dcache/src/main/java/org/dcache/pool/classic/IoQueueManager.java

```diff
@@ -451,16 +461,43 @@ public Serializable call() throws NoSuchElementException
                     comparator = comparator.reversed();
                 }
 
+                // Prepare filters
+                Predicate<MoverRequestScheduler.PrioritizedRequest> clientIpFilter;
+                if (clientHost != null && !clientHost.isEmpty()) {
+                    try {
+                        // check and resolve host
+                        clientHost = InetAddress.getByName(clientHost).getHostAddress();
+                    } catch (UnknownHostException e) {
+                        return String.format("Unknown host %s", clientHost);
```

Comment:
A better way of handling this is to throw a CommandException.  There are a few places where there's a distinction between a command exiting normally and abnormally (e.g., the colour of the output).